cmds/core/insmod/insmod_darwin.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Insert a module into the Linux kernel
+//
+// Synopsis:
```

Comment:
The bigger issue is that you have created files that claim to implement operations on darwin that do no exist. For example, there is no switch_root on darwin. That does not exist. Plus, you have files named _darwin.go that reference Linux. This just makes no sense. I appreciate the work but it's not solving the problem the right way.